chromadb/api/segment.py

```diff
@@ -893,6 +892,18 @@ def _get_collection(self, collection_id: UUID) -> t.Collection:
             )
         return collections[0]
 
+    @trace_method("SegmentAPI._scan", OpenTelemetryGranularity.ALL)
+    def _scan(self, collection_id: UUID) -> Scan:
+        collection_segments = self._sysdb.get_collection_with_segments(collection_id)
+        scope_to_segment = {segment["scope"]: segment for segment in collection_segments["segments"]}
+        return Scan(
```

Comment:
Nit:Worth commenting the assumption that we assume one segment per scope for now

Reply:
Added comments